index.bs

```diff
@@ -4446,6 +4655,54 @@ partial interface MLGraphBuilder {
     </pre>
 </div>
 
+<details open algorithm>
+  <summary>
+    To <dfn for=MLGraphBuilder>calculate pool output sizes</dfn> given {{MLInputOperandLayout}} |layout|, [=/list=] of 4 unsigned integers |inputShape|, {{MLRoundingType}} |roundingType|, [=/list=] of 2 unsigned integers |windowDimensions|, [=/list=] of 4 unsigned integers |padding|, [=/list=] of 2 unsigned integers |strides|, [=/list=] of 2 unsigned integers |dilations|, optional [=/list=] of 2 unsigned integers |outputSizes|, and {{MLAutoPad}} |autoPad|, perform these steps. They return a [=/list=] of 4 unsigned integers.
```

Comment:
Maybe be more explicit about pool2d? "calculate pool2d output sizes"

Reply:
done!